src/koans/01_equalities.clj

```diff
@@ -32,5 +32,8 @@
   "Symbolism is all around us"
   (= 'hello (symbol __))
 
+  "Introducing nil"
```

Comment:
Could be improved in my opinion, the plain  `Introducing nil` doesn't sound very consistent with the koan style of writing